Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -27,6 +27,15 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         sender.isEnabled = false
 
         let request = BTPayPalCheckoutRequest(amount: "4.30")
+        let lineItem = BTPayPalLineItem(
+                    quantity: "1",
+                    unitAmount: "5.00",
+                    name: "item one 1234567",
+                    kind: .credit)
```

Comment:
We can single line this
```suggestion
        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "5.00", name: "item one 1234567", kind: .credit)
```